toolchain/check/testdata/impl/no_prelude/impl_cycle.carbon

```diff
@@ -0,0 +1,2431 @@
+// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+// Exceptions. See /LICENSE for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
```

Comment:
Is this a test that benefits from including the generated SemIR, or should it be suppressed? It is quite long.

Reply:
As long as we're testing things like lookup through multiple impls elsewhere, we could suppress it - how do we do that?